app/frontend/src/features/messages/surfing/NewHostRequest.tsx

```diff
@@ -112,14 +117,16 @@ export default function NewHostRequest() {
             multiline
             fullWidth
           />
-          <Button
-            type="submit"
-            color="primary"
-            onClick={onSubmit}
-            loading={mutation.isLoading}
-          >
-            Send
-          </Button>
+          <div className={classes.buttonContainer}>
```

Comment:
use `Box`

Reply:
Should I replace every `buttonContainer` `div` with `Box`? I did not introduce it, seen this div wrapper in some component so I thought I will copy for consistency.

Reply:
Just change this one for now and we can decide what to do overall and fix the others later